crowbar_framework/app/models/node_object.rb

```diff
@@ -1451,6 +1451,31 @@ def disk_release!(device, owner)
     disk_release(device, owner) and save
   end
 
+  def verify_claimed_disks
+    unreferenced_disks = Array.new
+    crowbar_wall[:claimed_disks].each do |disk, _claim|
+      path, device_name = disk.split("/")[-2..-1]
+
+      unless disk =~ /^\/dev\/disk\//
+        unreferenced_disks.push(disk) if node[:block_device][device_name].nil?
+        next
+      end
+
+      if path == "cciss"
```

Comment:
What I meant to say is that the ```unless disk =~ /^\/dev\/disk\//``` block is going to be executed for disk with ```/dev/cciss/something``` in their name. I will not do any harm AFAICS, but I think is generally cleaner to execute the more specific ```if path == "cciss"``` block first.